src/fread.c

```diff
@@ -635,12 +635,12 @@ static void str_to_i32_core(const char **pch, int32_t *target, bool parse_date)
   }
 }
 
-static void StrtoI32(FieldParseContext *ctx)
+static void parse_i32(FieldParseContext *ctx)
```

Comment:
wouldn't the consistent naming here be `parse_int32_str` (`parse_${DEST_TYPE}_${SOURCE_DESCRIPTION}`)?